src/components/DepositWidget/Row.tsx

```diff
@@ -28,19 +31,39 @@ const TokenTr = styled.tr`
 
   &.selected {
     background-color: #ecdcff;
+
+  div.success {
+    color: #63ab52;
+    margin-top: -0.5rem;
   }
 `
 
 export interface RowProps {
-  tokenBalances: TokenBalanceDetails
+  initialTokenBalances: TokenBalanceDetails
```

Comment:
I wouldn't change the name. `tokenBalances` was fine. From the parent point of view he just hand the tokenBalances, then the child use them just to init them ones he can modify

Reply:
Reverted